graylog2-web-interface/src/views/components/searchbar/queryinput/custom-lucene-mode.js

```diff
@@ -91,7 +91,7 @@ ace.define(
           },
           {
             token: 'term',
-            regex: /[\w\\/]+/,
+            regex: /[\w\\/.]+/,
```

Comment:
As far as i can tell we do not need to escape the `.` in this case, because it is already treated as a "literal" and not a wildcard.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class#non-v-mode_character_class

"Non-v-mode character classes interpret most character [literally](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Literal_character) and have less restrictions about the characters they can contain. For example, `.` is the literal dot character, not the [wildcard](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Wildcard)."

Reply:
That's good to know, thanks for sharing this!